src/main/java/org/openelisglobal/method/daoimpl/MethodDAOImpl.java

```diff
@@ -180,24 +181,39 @@ public MethodDAOImpl() {
 //		}
 //	}
 
-//	@Override
-//	public List getAllMethods() throws LIMSRuntimeException {
-//		List list ;
-//		try {
-//			String sql = "from Method";
-//			org.hibernate.Query query = entityManager.unwrap(Session.class).createQuery(sql);
-//			list = query.list();
-//			// entityManager.unwrap(Session.class).flush(); // CSL remove old
-//			// entityManager.unwrap(Session.class).clear(); // CSL remove old
-//
-//		} catch (RuntimeException e) {
-//			// bugzilla 2154
-//			LogEvent.logError("MethodDAOImpl", "getAllMethods()", e.toString());
-//			throw new LIMSRuntimeException("Error in Method getAllMethods()", e);
-//		}
-//
-//		return list;
-//	}
+	@Override
```

Comment:
getAll and get (by Id) should already be implemented by the BaseDAOImpl that this class extends

Reply:
the same should be true of BaseServiceImpl